components/search_engines/brave_prepopulated_engines.cc

```diff
@@ -14,6 +14,13 @@ namespace TemplateURLPrepopulateData {
 // IMPORTANT! Make sure to bump this value if you make changes to the
 // engines below or add/remove engines.
 const int kBraveCurrentDataVersion = 29;
```

Comment:
do we have comments about bumping this version in the code where people typically change engines? It could be easy to miss

Reply:
Good call - let me add one!

Reply:
@iefremov updated! 😄